css/style.css

```diff
@@ -0,0 +1,68 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto');
+
+@font-face {
+    font-family: 'Glyphicons Halflings';
+    src: url('fonts/glyphicons-halflings-regular.eot');
+    src: url('fonts/glyphicons-halflings-regular.eot?#iefix') format('embedded-opentype'), url('fonts/glyphicons-halflings-regular.woff') format('woff'), url('fonts/glyphicons-halflings-regular.ttf') format('truetype'), url('fonts/glyphicons-halflings-regular.svg#glyphicons_halflingsregular') format('svg');
+}
+
+body {
+    background: #e0e0e0;
+    font-family: 'Roboto', sans-serif;
+}
+.container {
+    max-width: 1200px;
+    padding: 20px;
+}
+.phone-details {
+    max-width: 800px;
+    height: auto;
+    border: 3px solid gray;
+    display: flex;
+}
+.phone-info {
+    padding: 14px 0 14px 20px;
+    flex: 1.1;
```

Comment:
może zamiast używać wartości typu `1.1` da się to samo osiągnąć używając kombinacji typu:
```
flex: 1 0 auto
```
dla elementu trochę większego i 
```
flex: 0 1 auto
```
dla tych mniejszych. Nie jestem na 100% pewna czy zadziała

Jeśli dobrze rozumiem dokumentację (https://developer.mozilla.org/en-US/docs/Web/CSS/flex) to dodanie pojedynczej wartości manipuluje tylko `flex-grow`